### DISCUSSION TITLE
Ability to get not encoded member's email in `{exp:member:custom_profile_data}`

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3705] [Feature Requests]
### Suggest an idea for ExpressionEngine

```
{exp:member:custom_profile_data}
  {email}
{/exp:member:custom_profile_data}
```

`{email}` here returns a Javascript-encoded email address and there is no option to get a not encoded email address. It's kind of strange.

**Comment by intoeetive** [comment]
This makes sense. Encoding email is always possible with `{encode=}` tag, while opposite is not possible with built-in tools.

Going to change this is https://github.com/ExpressionEngine/ExpressionEngine/commit/3143a111bbcb2a591049d0276ba58be31768173e
